healthCheck/index.js

```diff
@@ -3,11 +3,26 @@
  * author : Aman Karki.
  * created-date : 01-Feb-2021.
  * Description : Health check Root file.
-*/
+ */
 
-let healthCheckService = require("./health-check");
+const healthCheckService = require('./health-check')
 
 module.exports = function (app) {
-    app.get("/health",healthCheckService.health_check)
-    app.get("/healthCheckStatus",healthCheckService.healthCheckStatus);
-}
\ No newline at end of file
+	app.get('/project/health', async (req, res) => {
```

Comment:
@MallanagoudaB don't change /health to /project/health instead adjut it in npm config source route